rs/sns/cli/src/lib.rs

```diff
@@ -62,17 +61,46 @@ pub struct CliArgs {
     #[clap(subcommand)]
     pub sub_command: SubCommand,
 
+    /// The user identity to run this command as. It contains your principal as well as some things DFX associates with it like the wallet.
+    #[arg(long, global = true)]
+    identity: Option<String>,
+
+    #[command(flatten)]
+    network: NetworkOpt,
```

Comment:
This changes the CLI format, right? Could add this, maybe with a short example command, in the PR description, please?

Reply:
Done!